leapp/utils/output.py

```diff
@@ -15,35 +18,55 @@ class Color(object):
     yellow = "\033[1;33m" if sys.stdout.isatty() else ""
 
 
-def pretty_block(string, color=Color.bold, width=60):
+def pretty_block_text(string, color=Color.bold, width=60):
     return "\n{color}{separator}\n{text}\n{separator}{reset}\n".format(
         color=color,
         separator="=" * width,
         reset=Color.reset,
         text=string.center(width))
 
 
+@contextmanager
+def pretty_block(text, target, end_text=None, color=Color.bold, width=60):
+    target.write(pretty_block_text(text, color=color, width=width))
+    target.write('\n')
+    yield
+    target.write('\n')
```

Comment:
```suggestion
```
Redundant blank line.

Reply:
Not sure if it is redundant, if I want a blank line before the pretty block

Reply:
The pretty block already starts with a blank line though.
It's not really a problem, I just find it weird to have two consecutive blank lines in the output.

Reply:
oh lol my bad, I overlooked that new line I will remove it